pkg/operation/seed/seed.go

```diff
@@ -393,6 +394,23 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 			return err
 		}
 
+		if conf.Logging != nil && conf.Logging.Loki != nil && conf.Logging.Loki.GardenLoki != nil &&
+			conf.Logging.Loki.GardenLoki.PriorityClassValue != nil {
+			priority := *conf.Logging.Loki.GardenLoki.PriorityClassValue
+			if err := deletePriorityClassIfValueNotTheSame(ctx, k8sSeedClient.Client(), common.GardenPriorityClassName, priority); err != nil {
+				return err
+			}
+			lokiValues["priorityClass"] = map[string]interface{}{
+				"value": priority,
+				"name":  common.GardenPriorityClassName,
+			}
+		} else {
+			pc := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: common.GardenPriorityClassName}}
+			if err := k8sSeedClient.Client().Delete(ctx, pc); client.IgnoreNotFound(err) != nil && !meta.IsNoMatchError(err) {
+				return err
```

Comment:
This will delete it on each seed bootstrap reconciliation if the loki config is not specified. Why?

Reply:
There is a case when after removing the configuration, there is no one to remove this PriorityClass and it will be removed only when the Logging stack is deleted.

Reply:
I got it. When the config is not set, there is nothing to use the `garden-loki` priority class, therefore it is not needed.